### ISSUE TITLE
(HTML) "<table>" tags should have a description

### ISSUE THREAD

**Issue by DanielfQo** [DanielfQo/CRUD-PYTHON#14]

**Comment by KevinRodriguezLima** [comment]
**Descripción:**
"<table>" tags should have a description" Para que las tablas sean accesibles para los usuarios con discapacidad visual, es importante que proporcionen una descripción de su contenido antes de que los datos sean accesibles. 

**Archivos:**

    my-app/templates/public/empleados/lista_empleados.html
    my-app/templates/public/usuarios/lista_usuarios.html
